presto-product-tests/bin/run_on_docker.sh

```diff
@@ -166,16 +166,16 @@ environment_compose logs --no-color -f ${EXTERNAL_SERVICES} &
 
 HADOOP_LOGS_PID=$!
 
-# wait until hadoop processes is started
-retry check_hadoop
-
 # start presto containers
 environment_compose up -d ${PRESTO_SERVICES}
 
 # start docker logs for presto containers
 environment_compose logs --no-color -f ${PRESTO_SERVICES} &
 PRESTO_LOGS_PID=$!
 
+# wait until hadoop processes is started
```

Comment:
`# ensure hadoop started properly` ?

Reply:
To make sure that `hadoop started properly` is not that easy comparing to what we did.